spec/inputs/integration/opensearch_spec.rb

```diff
@@ -54,8 +54,8 @@
   end
 
   describe 'against a secured opensearch', :secure_integration => true do
-    let(:user) { ENV['ELASTIC_USER'] || 'simpleuser' }
-    let(:password) { ENV['ELASTIC_PASSWORD'] || 'abc123' }
+    let(:user) { 'admin' }
+    let(:password) { 'admin' }
```

Comment:
why we dropped ability to set user/password and left only hardcoded values?

Reply:
We most commonly use 'admin'...'admin'. But, it is good to have the ability to set them, will add it back.